CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+## Thanks
+
+Thank you for considering contributing to **GrPPI. Our aim is to make **GrPPI** useful for the software comunity at large. 
+Any kind of contribution is highly appreciated.
+
+## Why should you read this guideline?
+
+Following this guideline will help you to keep contact with **GrPPI** team.
+This will result in shorter time to process your bug reports, suggestions and pull requests.
+
+## Ways of contributing to GrPPI
+
+You may contribute in a number of ways:
+
+* Filing bug reports.
+* Suggesting new features.
+* Contributing with documentation pull requests.
+* Contributing with pull requests with improved implementation of patterns or back ends.
+
+# Contributing go GrPPI
+
+## General rules
+
+* Please read and follow the GrPPI Code of Conduct.
```

Comment:
I think that this sentece would be better:

Please read and follow the GrPPI Code of Conduct before contributing in this project